tools/underwater-gps.py

```diff
@@ -60,6 +68,108 @@
                                       # E=Estimated, N=not valid, S=Simulator.
        + "*")                         # Checksum
 
+
+def report_status(*args):
+    """
+    reports the current status of this service
+    """
+    # Do not report the same status multiple times
+    global last_status
```

Comment:
Is it not better to move to a class ?

Reply:
it is, but this is not the time. this is a bugfix release.